.vscode/launch.json

```diff
@@ -1,38 +0,0 @@
-{
```

Comment:
Oh, I didnt realize this file was already part of the ASPECT repository. Please undo this change, you should be able to do this with `git checkout upstream/main .vscode/launch.json`